engine/classes/Elgg/Application.php

```diff
@@ -959,6 +960,7 @@ public function loadSettings() {
 	 */
 	private static function getEngineLibs() {
 		return [
+			'constants.php',
```

Comment:
If it's necessary to move this it means we're executing code where we shouldn't be.

Reply:
I was getting an error that `ACCESS_PUBLIC` is not defined during the test. I think the problem is PHPUnit bootstrap. I think it's trying to parse test files before actually letting the application bootstrap itself. Maybe adding loadCore will help